deploy-application-charts/action.yml

```
@@ -80,11 +80,9 @@ runs:
 
     - name: The public URL for a application deployed on a Kubernetes cluster.
       id: get-public-url
-      shell: bash
-      run: |
-        echo "public_url<<EOF" >> $GITHUB_OUTPUT
-        echo "$(yq '.applicationHosts' ${{ inputs.values_file }} || yq '.hosts' ${{ inputs.values_file }})" | while read line; do
-          echo $line | awk '{print $2}' >> $GITHUB_OUTPUT
-        done
-        echo "EOF" >> $GITHUB_OUTPUT
-
+      uses: mikefarah/yq@master
```

```suggestion
      uses: mikefarah/yq@4
```

不要用 master，如果最新代码出问题了，会影响我们的 action

---

cc @moonRider

---

好，之前前端打包的时候让把，checkout 和其他的 acitons 改成最新的，我以为这样更好

---

https://github.com/mikefarah/yq/tree/v4

---

ok